emails/tests/cleaners_tests.py

```diff
@@ -417,24 +409,3 @@ def test_many_profile_detector_with_users() -> None:
     One or no Profile: 4 (100.0%)
     Many Profiles    : 0 (  0.0%)"""
     assert report == expected
-
-
-@pytest.mark.django_db
-def test_many_profile_detector_with_problems() -> None:
```

Comment:
question (non-blocking): should we remove this BEFORE we're enforcing single profiles with `OneToOneField`?

Reply:
Yes, we have to remove this test. The logic to test for many profiles remains for now, so `./manage.py cleanup_data --many-profiles` still works and will detect users with multiple profiles. However, the test setup will fail once `OneToOneField` is applied, so the migrations tests will fail. This (and `baker.make(Profile)`) blocked this from being a one-deployment change.